metis-normalization/src/main/java/eu/europeana/normalization/dates/edtf/InstantEdtfDate.java

```diff
@@ -101,6 +101,7 @@ && getEdtfDatePart().getYear() < THRESHOLD_4_DIGITS_YEAR) {
           firstDay.getEdtfDatePart().setDay(1);
         }
       }
+
       // TODO: 25/07/2022 What about > THRESHOLD_4_DIGITS_YEAR??
```

Comment:
The part where > THRESHOLD_4_DIGITS_YEAR is not possible because it's in the future, so we don't have to check it.
Verify though that the contents of this class are always considered valid before the call of this method.